heartbeat/azure-events.in

```diff
@@ -221,7 +222,7 @@ class clusterHelper:
 
 		nodes = []
 		nodeList = clusterHelper._exec("crm_node", "--list")
-		for n in nodeList.split("\n"):
+		for n in nodeList.decode().split("\n"):
```

Comment:
We could replace that `split("\n")' by `splitlines()`.

The difference is that if the splitted string is emtpy, the first option returns `[""]` and the second `[]`.
I guess it wouldn't have any impact in the behaviour though